### PR TITLE
Adding blog post announcement for pulp_ansible beta

### PR DESCRIPTION
**Pull request by daviddavis**

### CODE REVIEW

File: _posts/2018-04-27-pulp_ansible-beta-release.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+title: Pulp Ansible Plugin Beta 1 Release
+author: David Davis
+tags:
+  - release
+  - 3.0
+  - pulp_ansible
```
Comment (bmbouter) [inline _posts/2018-04-27-pulp_ansible-beta-release.md]: This tag won't work. You have to do a few small things to 'add a tag' See this dir if you want to do that:  https://github.com/pulp/pulpproject.org/tree/gh-pages/tags

**Comment by bmbouter** [comment]
I'm uploading the demo to YouTube, does embedding it in this post make any sense?

File: _posts/2018-04-27-pulp_ansible-beta-release.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+out the [pulp_ansible repository on Github](https://github.com/pulp/pulp_ansible).
+
+## Getting involved
+
+To view our progress, report bugs, or request new features, check out our issue tracker at
+[https://pulp.plan.io/projects/ansible_plugin](https://pulp.plan.io/projects/ansible_plugin).
+
+## Demo video
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/EnD8BGcNcjU" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
```
Comment (daviddavis) [inline _posts/2018-04-27-pulp_ansible-beta-release.md:38]: Supposedly this should work according to https://stackoverflow.com/a/10530266

Comment (bmbouter) [reply _posts/2018-04-27-pulp_ansible-beta-release.md:38]: Yeah it will, it's the same syntax I use in the embedded community demos.

**Review by bmbouter — APPROVED** [review]
This looks great, let's announce!

**Pull request merged**